tests/rules/property.js

```diff
@@ -623,8 +673,28 @@ eslintTester.run("property", rule, {
                     type: "AssignmentExpression"
                 }
             ],
+            options: [
+                {
+                    variableTracing: true
+                }
+            ],
             parserOptions: { ecmaVersion: 6 }
         },
+        {
+            code: "let msg = '<b>safe</b>'; let altMsg = 'also cool';  if (monday) { msg = altMsg; }; y.innerHTML = msg;",
+            parserOptions: { ecmaVersion: 6 },
+            errors: [
+                {
+                    message: /Unsafe assignment to innerHTML/,
+                    type: "AssignmentExpression"
+                }
+            ],
+            options: [
+                {
+                    variableTracing: 0
```

Comment:
Nit, `variableTracking: false` (doesn't matter in practice, it would just for consistency with the boolean value expected and set elsewhere).